src/game/items/CItem.cpp

```diff
@@ -1901,7 +1901,7 @@ bool CItem::SetName( lpctstr pszName )
 	return SetNamePool( pszName );
 }
 
-HUE_TYPE CItem::GetHue() const  //Override of CObjBase::GetHue()
+HUE_TYPE CItem::GetHue_visualforclient() const
```

Comment:
Can you change the method name with a shorter and less "dispersive" one? Like GetHueVisible (which also respects the naming convention with the uppercase letters).